zulipterminal/config/keys.py

```diff
@@ -220,7 +220,7 @@ class KeyBinding(TypedDict):
         'key_context': 'stream_view',
     },
     'ALL_MESSAGES': {
-        'keys': ['a', 'esc'],
+        'keys': ['a'],
```

Comment:
This is a temporary change. To make the linting tests pass.
As both the `Esc` key mappings clash in the 'general' context.

I could add an exception instead too, if necessary later.
Or I could move the `ALL_MESSAGES` command to the "message_view" context instead until that bug is resolved.